vSphereHypervisorCollector.class.inc.php

```diff
@@ -94,23 +99,23 @@ public static function GetHypervisors()
 
 				Utils::Log(LOG_DEBUG, "Server {$oHypervisor->name}: {$oHypervisor->hardware->systemInfo->vendor} {$oHypervisor->hardware->systemInfo->model}");
 				Utils::Log(LOG_DEBUG, "Server software: {$oHypervisor->config->product->fullName} - API Version: {$oHypervisor->config->product->apiVersion}");
-				
+
 				$aHypervisorData = array(
 						'id' => $oHypervisor->getReferenceId(),
 				        'primary_key' => $oHypervisor->getReferenceId(),
 				        'name' => $oHypervisor->name,
 						'org_id' => $sDefaultOrg,
 						'brand_id' => $oBrandMappings->MapValue($oHypervisor->hardware->systemInfo->vendor, 'Other'),
 						'model_id' => $oModelMappings->MapValue($oHypervisor->hardware->systemInfo->model, ''),
-						'cpu' => $oHypervisor->hardware->cpuInfo->numCpuPackages,
+						'cpu' => $oHypervisor->hardware->cpuInfo->$sVMCPUAttribute,
```

Comment:
Please add a check if the field actually exists, otherwise this will introduce php warnings when wrongly configured..

Reply:
Done